src/components/ContentReport/ContentReportForm.vue

```diff
@@ -33,15 +33,16 @@
         </div>
       </fieldset>
 
-      <span class="caption has-text-weight-semibold has-text-grey">
+      <p class="caption has-text-weight-semibold has-text-grey margin-bottom-normal">
         For security purposes, CC collects and retains anonymized IP
         addresses of those who complete and submit this form.
-      </span>
+      </p>
 
       <button type="button"
-              :disabled="selectedReason === null"
-              class="button next-button tiny is-info margin-top-normal is-pulled-right"
-              @click="onIssueSelected()">
+         :disabled="selectedReason === null"
+         :class="['button', 'next-button', 'tiny', 'is-info', 'is-pulled-right',
```

Comment:
please revert the syntax to just plain string. The indentation is also wrong

Reply:
Done